### DISCUSSION TITLE
Reddit - Videos

### DISCUSSION THREAD

**Discussion by LukeG20** [FreshRSS/FreshRSS#4609] [Q&A] [answered]
I can see there's plugins to allow video playback from youtube but is there something similar for Reddit?

**Comment by aledeg** [comment, marked as answer]
Maybe this will do https://github.com/aledeg/xExtension-RedditImage

**Reply by LukeG20** [reply]
That seems perfect I'll give it a try this afternoon. I did see it on https://github.com/FreshRSS/Extensions but disregarded it due to the name.

Thanks for the response.
